notice_and_comment/templates/regulations/comment-disclaimer.html

```diff
@@ -1,21 +1,23 @@
 <div>
-  <ul>
+  <ul class="statement-list">
     <li>
-      <h4>You are filing a document into an official docket.</h4>
+      <h6>You are filing a document into an official docket.</h6>
       <div id="disclaimer-1">
```

Comment:
As those existed before this PR, I created #275 to address them. I'd prefer to see the incremental progress.